### DISCUSSION TITLE
Navigateur Dans Synapps

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#640] [Q&A]
Bonjour,

J'ai un client qui souhaite accéder aux autres automates avec un synapps dans chacun depuis un automate maitre

En gros, tous les automates communiquent en 4G vers un Automate. 

Le but est d'utiliser la 4G du maitre pour aller vers les esclaves. Hors, je crois que l'acteur Iframe etc... n'utilise pas le support 4G en priorité. 

![image](https://github.com/witsa/synapps/assets/46195964/00ccf8c1-54be-45d4-a7a1-5a0c1d6d0f16)

Merci,
Flo

**Comment by Raphwit** [comment]
Bonjour, si le client dispose des adresses de tous ses automates il suffit de changer l'adresse de l'automate dans l'acteur l'iFrame pour accéder à l'interface de l'automate qu'il souhaite.
Il faut donc avoir accès à ces automates pour faire cela !

**Reply by FlorentAdell** [reply]
Bonjour, j'ai exposé mon problème à Nico, mais malheureusement ça pose des soucis de sécurité donc impossible de pourvoir accéder aux réseaux sous forme d'un "routage" ou "bridge", donc les synoptiques ne sont pas dispo
